figures/time/example_lomb_scargle.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
```

Comment:
Always put a docstring at the top of Python scripts, with at least one line about what it does. In this case:
```
"""
Plot example Lomb-Scargle figures for the Gammapy docs at
http://docs.gammapy.org/en/latest/time/period.html
"""